### PR TITLE
Separate C7 and C8 Rules

### PR DESCRIPTION
**Pull request by philippfromme**
Requires https://github.com/bpmn-io/bpmnlint/pull/99.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
   "license": "MIT",
   "devDependencies": {
     "bpmn-moddle": "^7.1.3",
-    "bpmnlint": "^7.8.0",
```
Comment (nikku) [inline package.json:28]: Is it intentional to move this to a production dependency?

Comment (philippfromme) [reply package.json:28]: Ooops!

Comment (philippfromme) [reply package.json:28]: Fixed!

**Comment by nikku** [comment]
Looks good. Took me a little bit to grasp what we fixed with https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/87/commits/0c675bc9a3e7f44ef28727fee04e49145a687ad7 :wink:.

**Comment by philippfromme** [comment]
> Took me a little bit to grasp what we fixed with [0c675bc](https://github.com/camunda/bpmnlint-plugin-camunda-compat/commit/0c675bc9a3e7f44ef28727fee04e49145a687ad7) 😉.

Everything is what we fixed! But mainly the fact that we only care about executable processes and some conventions.

**Pull request merged**